share/rcm.sh.in

```diff
@@ -147,6 +147,20 @@ de_dot() {
   echo "$1" | sed -e "s|$DEST_DIR/||" | sed -e 's/^\.//'
 }
 
+DELIMITER="\a"
+
+encode() {
+  local file="$1"
+
+  echo "$file" | sed -e "s/ /$DELIMITER/g"
```

Comment:
Consider using `ts` for this. Something about this looks a bit wrong. 
See the example below with the `sed` used here vs. `ts`. 

```sh
sh-3.2$ DELIMITER="\a"
sh-3.2$ \
> encode() {
>   local file="$1"
>   echo "$file" | sed -e "s/ /$DELIMITER/g"
> }
sh-3.2$ \
> decode() {
>   local file="$1"
>   echo "$file" | sed -e "s/$DELIMITER/ /g"
> }
sh-3.2$ echo $(decode $(encode "this is a test"))
this is test
sh-3.2$ \
> encode() {
>   local file="$1"
>   echo "$file" | tr " " "$DELIMITER"
> }
sh-3.2$ \
> decode() {
>   local file="$1"
>   echo "$file" | tr "$DELIMITER" " "
> }
sh-3.2$ echo $(decode $(encode "this is a test"))
this is a test
sh-3.2$
```